app/assets/stylesheets/forms.css.scss

```diff
@@ -0,0 +1,67 @@
+@import 'shared';
+
+.form-header {
+  width: 100%;
+}
+
+.form-title {
+  border-bottom: 4px solid $theme-black;
+  padding-left: 0;
+  padding-right: 0;
+  h1 {
+    text-transform: uppercase;
+    font-weight: bold;
+    font-family: 'Oswald';
+    font-size: 24px;
+    padding-bottom: 10px;
+  }   
+}
+
+.form-fields {
```

Comment:
This whole file is mostly copied from https://github.com/heatseeknyc/heatseeknyc/blob/master/app/assets/stylesheets/settings.css.scss
since the elements are different from the users settings which uses DL to wrap each of the label/input groups and formtastic doesn't use DLs.  This would replace settings.css, which can be deleted if User settings forms are converted to Formtastic.